aedttest/simulation_data.py

```diff
@@ -113,7 +113,11 @@ def parse_profile_file(profile_file, design_name, variation, setup_name):
             if "elapsed time" in line.lower():
                 elapsed_time = line.lower()
             if "cells" in line.lower():
-                cell_number = int(line.lower().split()[-2])
+                cell = [x for x in line.strip().lower().split(",") if "cell" in x]
+                cell_number = cell[0].replace("total", "")
+                cell_number = cell_number.replace("cells", "")
```

Comment:
Apply regex

Reply:
![image](https://user-images.githubusercontent.com/93125854/174261731-d807fcca-486c-45b8-866d-80843ff992d4.png)
![image](https://user-images.githubusercontent.com/93125854/174261856-7901bf52-9f17-4398-ab5f-3de6eb42860d.png)

any good idea to have regex for the ones with "cell"